desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -443,9 +444,27 @@ class OnePageViewModel {
     };
 
     const openImporter = function (path) {
+      let databasename = '';
+      let interpreter = '';
+      const table = path.substring(path.lastIndexOf(':') + 1, path.lastIndexOf(';'));
       self.loadApp('importer');
+      huePubSub.publish(ASSIST_GET_SOURCE_EVENT, source => {
+        interpreter = source;
+      });
+      huePubSub.publish(ASSIST_GET_DATABASE_EVENT, {
+        connector: { id: interpreter },
+        callback: databaseDef => {
+          databasename = databaseDef.name;
+        }
+      });
       self.getActiveAppViewModel(vm => {
         vm.createWizard.source.path(path);
+        vm.currentStep(2);
+        vm.createWizard.source.interpreter(interpreter);
+        vm.createWizard.destination.name(databasename + '.' + table.substring(0, table.length - 4));
+        window.setTimeout(() => {
+          vm.createWizard.indexFile();
+        }, 500);
```

Comment:
So here we are using 500 ms because we were using throttle = 500 in destination.name in importer.mako. So we have to wait 500ms. https://github.com/cloudera/hue/blob/master/desktop/libs/indexer/src/indexer/templates/importer.mako#L2213